pkgs/by-name/yu/yunfaavatar/package.nix

```
@@ -0,0 +1,42 @@
+{ lib
+, stdenvNoCC
+, fetchFromGitHub
+, bash
+, makeWrapper
+, pciutils
+, x11Support ? true
+, ueberzug
+, fetchpatch
+, pkgs
+}:
+stdenvNoCC.mkDerivation rec {
+  pname = "yunfaavatar";
+  version = "0.1.0";
+
+  src = fetchFromGitHub {
+    owner = "yunfachi";
+    repo = "yunfaAvatar";
+    rev = version;
+    hash = "sha256-JbolO1I4K9+RBFJ/U611En4Rx5cG2+z1IcxYEeiaq6Q=";
+  };
+
+  outputs = [ "out" "man" ];
+
+  strictDeps = true;
+  buildInputs = [ bash pkgs.imagemagick ];
+  nativeBuildInputs = [ makeWrapper ];
+
+  makeFlags = [
+    "PREFIX=${placeholder "out"}"
+    "SYSCONFDIR=${placeholder "out"}/etc"
+  ];
+
+  meta = with lib; {
+    description = "Utility for automatic centralized changing of avatar in Github, Discord, Steam, Hypixel, etc.";
```

Descriptions can't end in a `.`, I know this is an edge case, but to make your life easier you could just rewrite it like this e.g.:
```suggestion
    description = "Utility for automatic centralized changing of avatar in Github, Discord, Steam, Hypixel, and many more";
```